tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
```

Comment:
I think if we set this to `src` then we will be able to skip the `src` directory in the output, so our output is like `dist/index.js`

Reply:
done - this did require moving `test` into `src` btw